enhancements/audit-command.md

```diff
@@ -0,0 +1,843 @@
+
+#### As an admin and or maintainer of OLM index catalog, I would like to use audit to only perform the checks and get its result on JSON format so that, I can use it in CI job.
+
+To address this requirement user are able to use the flags `--output=JSON` and the `--checks-only`.
+
+Also, see that in this case user might only get the errors and not the warnings the flag `--errors-only` can also be used to address this need.
+
+#### As an admin and or maintainer of OLM index catalog, I'd like to check the operator bundles against to specific validators so that, I can easily decide what check is more appropriate for my scenario.
+
+To address this need users will be able to use the flag `--list-optional-validators` which is the same provide by SDK in order to check the available options, see:
+
+```
+$ operator-sdk bundle validate --list-optional
+NAME           LABELS                     DESCRIPTION
+operatorhub    name=operatorhub           OperatorHub.io metadata validation
+               suite=operatorframework    
+
+```
+
+Also, they will be able to select the available suite tests by using the flag `--select-optional` which also is equivalent the same option in SDK.
+
+#### As an admin and or maintainer of OLM index catalog, I'd like to be able to disable the validations and/or checks done for the operator bundles so that, I can easily decide what check is more appropriate for my scenario.
+
+To address this need users will be able to use the flags:
+
+- `--scorecard=false`: to disable the scorecard checks.
+- `--validation=false`: to disable the scorecard checks.
+
+#### As an admin and or maintainer of OLM index catalog, I would like to use audit to only audit the head bundle operators so that, I can have a faster result.
+
+To address this requirement user are able to use the flag `--head-only`.
+
+#### As an admin and or maintainer of OLM index catalog, I would like to define what is the tool which will be used to pull and save the images so that, I can define it accordingly with the tool which I have installed and has my credentials when required.
+
+To address this requirement user are able to use the flag `--container-tools` and inform the option as please them.
+
+#### As an admin and or maintainer of OLM index catalog, I'd like to define a timeout so that, I can customize it according to my expectations and avoid a process get stuck or increase the default time when I know that it will require more time to be processed.
+
+To address this requirement user are able to use the flag `--wait-time` and inform a time in minutes which will be used instead of the default timeout.
+
+#### As an admin and or maintainer of OLM index catalog, I'd like to look at all the bundles of a package holistically and attempts to detect potential error conditions so that, I can easily identify errors in the upgrade graph paths.
+
+//TODO:
+
```

Comment:
Thoughts: 
- Is the channel head and has replace for a bundle that does not exist then should return an error
- WIP